contracts/infrastructure/WalletFactory.sol

```diff
@@ -267,16 +277,12 @@ contract WalletFactory is Owned, Managed {
     {
         bool success;
         (success, _res) = _wallet.call(abi.encodeWithSignature("invoke(address,uint256,bytes)", _to, _value, _data));
-        if (success && _res.length > 0) { //_res is empty if _wallet is an "old" BaseWallet that can't return output values
-            (_res) = abi.decode(_res, (bytes));
-        } else if (_res.length > 0) {
+        if (!success) {
```

Comment:
You still need ` (_res) = abi.decode(_res, (bytes));` here.

Reply:
Ignoring this comment as we are now ignoring the returned bytes.